tests/upgradePMM_test.js

```diff
@@ -514,14 +526,7 @@ if (versionMinor >= 32) {
       const schedule = {
         service_id,
         location_id: locationId,
-        cron_expression: '*/20 * * * *',
```

Comment:
Well, I don't mind. I've just thought it does not deserve s class level constant because there is no prosperity on reusing. But still it's a test hardcode which is also goo to have in variable. So that's only could be judged on the personal taste of code organizing.